src/intentLogic/latestPodcast.js

```diff
@@ -53,4 +49,6 @@ const generatePodcastSpeech = (results, isNewIntent) => {
   return ack + buildLatestPodcastSpeech() + podcastTitles + followupQuestion()
 }
 
-const removePodcastFromTitle = (title) => title.replace(/(-|–)?\s?podcast$/, '')
+const getPodcastSeriesName = (tags) => {
+  return tags.filter(tag => tag.podcast).map(t => t.webTitle.split(/[-–]/)[0])
```

Comment:
Semantically speaking maybe, but there's only one `podcast` field that can be returned by capi anyway...